### ISSUE TITLE
Show multiple images in post and cards

### ISSUE THREAD

**Issue by AndreasWintherMoen** [BarkApp/bark#306]
Most of the posts have more than 1 image, but we only display the first. The post screen should have a carousel similar to the wiki screen. This can also be done for the cards, at least BigCard (probably not for smaller, because they are often in a horizontally scrolled list).

**Comment by haakonjacobsen** [comment]
I agree, just make a reusable component from the wiki page if it's not already made. We also need to hook this up with shared elements in the future.
